src/main/java/seedu/address/model/policy/EducationPolicy.java

```diff
@@ -30,11 +33,13 @@ public EducationPolicy() {
      * @param expiryDate the date of Policy's expiry.
      *                   Use null to initialize this policy with the default expiryDate.
      */
-    public EducationPolicy(PremiumAmount premiumAmount, CoverageAmount coverageAmount, ExpiryDate expiryDate) {
+    public EducationPolicy(PremiumAmount premiumAmount, CoverageAmount coverageAmount, ExpiryDate expiryDate,
+                           ClaimList claims) {
         super(
                 premiumAmount == null ? DEFAULT_PREMIUM_AMOUNT : premiumAmount,
                 coverageAmount == null ? DEFAULT_COVERAGE_AMOUNT : coverageAmount,
-                expiryDate == null ? new ExpiryDate(LocalDate.now().plus(DEFAULT_EXPIRY_DATE_PERIOD)) : expiryDate);
+                expiryDate == null ? new ExpiryDate(LocalDate.now().plus(DEFAULT_EXPIRY_DATE_PERIOD)) : expiryDate,
+                claims);
```

Comment:
should check if claims is null. If it is null, instantiate an empty claimlist. Same for Health and Lifepolicy.